### PR TITLE
Leaderboard: Part 2 hit the API

### PR DESCRIPTION
**Pull request by ivaan1992**
For this project I:

- Installed webpack.
- Installed linters.
- Created HTML file.
- Created CSS file.
- Created Script file.
- Used ES6.
- Implemented API

### CODE REVIEW

File: src/fetch.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export default function request(url, { method, body }) {
```
Comment (bimbolabuari) [inline src/fetch.js]: According to ES6 standard, let's make use of arrow functions instead of the` function` keyword ✊

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @ivaan1992 , 

You have done a good job so far 🚀  

## Highlights 🎉 

- Use `async` and `await` with the `fetch` method to send and receive data.

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: src/index.js [hunk 2]
```diff
@@ -1,22 +1,76 @@
...
+    body: JSON.stringify({ name }),
+  });
+  const { result } = await response.json();
+  gameID = result.split(' ')[''];
+  gameForm.remove();
+
+  toastr.success(result);
+});
+
+refresh.addEventListener('click', async (e) => {
```
Comment (Mihndim2020) [inline src/index.js:44]: - [x] Thank for you making the change requested by the former code reviewer, but after submitting a score and clicking on the refresh button, I don't see the score display, please kindly fix that, as it is a requirement for this project.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @ivaan1992, 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there

 # Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 3]
```diff
@@ -35,7 +35,7 @@ game.addEventListener('click', async (e) => {
     body: JSON.stringify({ name }),
   });
   const { result } = await response.json();
-  gameID = result.split(' ')[''];
+  gameID = result.split(' ').find((str, index) => index === 3);
```
Comment (bimbolabuari) [inline src/index.js:38]: As the previous code reviewer has pointed out 
> Thank for you making the change requested by the former code reviewer, but after submitting a score and clicking on the refresh button, I don't see the score display, please kindly fix that, as it is a requirement for this project.

This problem still persists, Kindly fix this before merging 🙂

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @ivaan1992 🥇,

Your project is almost complete!  Kindly fix the required changes, then... it's time to merge it :shipit:
 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 3 above)
Comment (ivaan1992) [reply src/index.js:38]: I just fix it!!! 
But I'll check what happens!! 

Captura de pantalla de 2021-12-18 14-25-11

**Pull request merged**